save-preprocessor/src/main/kotlin/com/saveourtool/save/preprocessor/service/TestDiscoveringService.kt

```diff
@@ -52,18 +56,22 @@ class TestDiscoveringService(
         log.info { "Starting to save new test suites for root test config in $repositoryPath" }
         return Mono.just(repositoryPath)
             .map { it.resolve(testRootPath) }
-            .map { getRootTestConfig(it.absolutePathString()) }
+            .flatMap { blockingToMono { getRootTestConfig(it.absolutePathString()) } }
```

Comment:
Combine previous steps to a single map:
```suggestion
            blockingToMono { getRootTestConfig(repositoryPath.resolve(testRootPath).absolutePathString()) }
```